plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -12,6 +12,18 @@ import ServiceConfigUtil from '../utils/ServiceConfigUtil';
 import ServiceConfigBaseSectionDisplay from './ServiceConfigBaseSectionDisplay';
 import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
 
+const getNetworkType = (networkType, appDefinition) => {
```

Comment:
Even though that it's more a stylistic question than actually an issue in this case,  I think that we should revisit our decision to allow arrow functions (actually only in cases where one want to "bind" `this`) and prohibit the use of arrow functions again, after reading through the Stack Overflow discussion.
Let's discuss this in our next team sync and decide wether we want to allow or prohibit the use and adjust our linting rules accordingly. 

/cc @dcos/frontend